teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown')
+    super(age, name)
```

Comment:
- [ ] Kindly add parent permission to initialize and super constructors.

Reply:
Thanks so much for the Review and remark, @sja-thedude.  Your effort and suggestion is taken dearly. 

However, The requirement for the Teacher class is as stated below, 

Create a class Teacher with the following:
    Inherits from Person.
    The constructor extends the parent's constructor by adding @specialization and a parameter for it.
    Override can_use_services? so it always returns true.

Your suggestion demands that I should add the permission to initialize, thus, 

1. Firstly, I do not understand the permission you are referring to, Please kindly share the syntax, 
2. I extended the parent class and declared the super constructor under the default's initialize method. 

Please kindly elaborate on the case and share idea. 

Thanks.

Reply:
- [ ] I think here you should add it as you add it above in student class so kindly add it as the previous code reviewer asked 😊🚀💯🔥 because both classes are inheirted from the baseclass person. 💯💯

>  def initialize(specialization, age, name = 'unknown', parent_permmission: true)
    super(age, name, parent_permmission)